website/docs/reference/deploy/import-export.md

```diff
@@ -9,6 +9,12 @@ The import and export API first appeared in Unleash 3.3.0.
 
 :::
 
+:::caution Deprecation notice
+
+Api admin state is deprecated from version 5. We recommend using `/api/admin/features-batch` instead for [import](https://docs.getunleash.io/reference/api/unleash/import-toggles) and [export](https://docs.getunleash.io/reference/api/unleash/export-features).
```

Comment:
Have we not documented the new features-batch ? I'd prefer having a link to that documentation if we have :)

Reply:
No, there's no documentation yet, I'd also prefer that TBH

Reply:
Will merge now, but also cc @kwasniew @andreas-unleash for awareness

Reply:
It's already done long time ago: https://docs.getunleash.io/reference/deploy/environment-import-export

Reply:
Thanks I couldn't find it, I'll make another PR to include it